contracts/dezswap_pair/src/contract.rs

```diff
@@ -363,6 +373,8 @@ pub fn provide_liquidity(
         }
     }
 
+    assert_slippage_tolerance(&slippage_tolerance, &deposits, &pools)?;
```

Comment:
I guess this can be confirmed by checking if `remain_amount` is greater than `slippage_tolerance * deposit` in for loop.

Reply:
https://github.com/dezswap/dezswap-contracts/pull/14/commits/6fcddf721189f470101a950f7e966310d73cd820